models/balancer/polygon/balancer_v2_polygon_lbps.sql

```diff
@@ -0,0 +1,119 @@
+{{
+    config(
+        schema = 'balancer_v2_polygon',
+        alias='lbps',
+        unique_key = ['pool_id'],
+        post_hook='{{ expose_spells(\'["polygon"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["stefenon"]\') }}'
+    )
+}}
+
+{% set create_lbp_contracts = [
+                                source('balancer_v2_ethereum', 'LiquidityBootstrappingPoolFactory_call_create'),
```

Comment:
looks like copy/paste error, having wrong blockchain here :)

Reply:
same in a few places throughout

Reply:
just solved the wrong chain issues.